kubernetes_deploy/production/ingress.yml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: laa-cla-public
   namespace: laa-cla-public-production
 spec:
+  tls:
+  - hosts:
+    - laa-cla-public-production.apps.cloud-platform-live-0.k8s.integration.dsd.io
```

Comment:
🙋‍♂️ We have checklegalaid.service.gov.uk as a public domain as well. Could we add that to the list, please?

Reply:
We don't have checklegalaid.service.gov.uk in the ingress rules, should we still add this under the tls hosts

Reply:
As far as I understand, we need to put the hosts needing TLS certificates here. Good question!
What do you think, @alkar?

Reply:
I'd say you could leave it like this for now and we can add the service.gov.uk domains when we get to do the certs as well and update the Ingress hosts. This change we asked for is only to ensure a smooth transition onto the new nginx-ingress setup we have.

Reply:
I see, thank you. Do we anticipate any change in the configuration?

Reply:
The future changes needed are similar to what we've done in https://github.com/ministryofjustice/fala/pull/32. If you'd like to go ahead and make these changes now, could you drop us a request to update the [config on our side](https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cloud-platform-components/cert-manager-external.tf#L4-L6)?

Reply:
@sldblog I have added checklegalaid.service.gov.uk to ingress rules and as a tls host